### ISSUE TITLE
Home page screen of the website

### ISSUE THREAD

**Issue by Ibrahim2750mi** [Ibrahim2750mi/tireless-timingila#2]
The home page of website could be like this
![home page screen](https://user-images.githubusercontent.com/74553450/180836124-c438cdfc-ed96-46c3-9d6a-491a237b5ae5.png)

We could use a font style like this for the screen
![font](https://image.freepik.com/free-vector/colorful-cartoon-font-effect-game-title_36662-326.jpg)

Open to suggestions what do you guys think? Also who would be able to do it?

**Comment by thatbirdguythatuknownot** [comment]
I don't know how to code in frontend, but the image placements look OK.

**Comment by Ibrahim2750mi** [comment]
https://www.fontspace.com/dilo-world-font-f72650 A font looking similar to the one in picture

**Comment by rustyxlol** [comment]
https://user-images.githubusercontent.com/27452933/181250283-0e205158-0802-408f-9d28-701feed3687f.mp4

I was thinking something like this

**Comment by Ibrahim2750mi** [comment]
> I was thinking something like this

The theme and layout looks good but the usage of buttons in the main menu I don't like, can't we just keep it simple like clicking on play then entering the name and then directly joining in a random games like slither.io or littleaclhemy? Or we can make a poll.

**Comment by rustyxlol** [comment]
yeah that's how it works, clicking play just throws you in a game

**Comment by Ibrahim2750mi** [comment]
> yeah that's how it works, clicking play just throws you in a game

Ok. Will make changes to make this the exact copy, do you have like a cool background idea? like gray background with slanting stripes?
